W-16993325-pce4-documentation-kt (#344)
mulesoft/docs-private-cloud: modules/ROOT/pages/operating-about.adoc

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: restarting-a-node.adoc

After your installation is complete, your instance of Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) is ready to use. 

== Users Using Anypoint Platform PCE

> Users registered in your external identity service should have access to your Anypoint Platform xref:access-management::organization.adoc[organization]. As an administrator, you can add xref:access-management::roles.adoc[roles] to these users.

Reviewer: ```suggestion
Ensure that users registered in your external identity service have access to your Anypoint Platform xref:access-management::organization.adoc[organization]. As an administrator, you can add these users to xref:access-management::teams.adoc[teams] to grant them access.
```
Roles are deprecated, please confirm if this suggestion is accurate.
